example/src/components/ParallaxVideo.tsx

```diff
@@ -44,7 +44,7 @@ const ParallaxVideo = ({ source, zIndex }: any) => {
 
   return (
     <Animated.View style={[styles.container, layerStyle]}>
-      <TransparentVideo source={source} style={styles.video} />
+      <TransparentVideo source={source} style={styles.video} autoplay />
```

Comment:
My knowledge of tsx is somewhat limited. In this context, where is `autoplay` coming from?

Reply:
ah, this is only the example. But basically in this case, putting autoplay here is essentially the equivalent of saying `autoplay={true}` (as compared to `autoplay={autoplay}`